releasenotes/notes/amplitude-estimation-refactor-92dd0a2905b6fa0e.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Add the refactored Amplitude Estimation algorithms.
+    The algorithms,
+    * ``AmplitudeEstimation`` (Brassard et al.)
+    * ``MaximumLikelihoodAmplitudeEstimation`` (Suzuki et al.)
+    * ``IterativeAmplitudeEstimation`` (Grinko et al.)
+    * ``FasterAmplitudeEstimation`` (Nakaji)
+    are separated from the problem-specific information, such as the :math:`\mathcal{A}`
+    and :math:`\mathcal{Q}` operators. This information is encapsulated in an
+    ``EstimationProblem``. Each of the algorithms is thus re-usable for different problems and
+    different algorithms can easily be compared on the same problem.
```

Comment:
Do we need a release note here? We're going to write one big release note covering the migrated aqua code and also have a dedicated migration guide (see #5493 ) so won't this be covered by that? My thinking is we've not had amplitude estimation algorithms in terra before so saying we've refactored algorithms doesn't make sense in the terra release note context because when we release it will be the first versions.

Reply:
Removed 👍🏻